public/index.js

```diff
@@ -112,18 +121,23 @@ dataSubmit.click((e)=>{
           console.log(urls, 'in matchFolder line 112')
           let shortUrl = urls[urls.length-1].shortened_url
           $('.url-list').append(`<div class= 'appended-url'>
-                                  <div>
-                                    <h4>Title: </h4>
-                                    <p>${urls[urls.length-1].title}</p>
-                                  </div>
-                                  <div>
-                                    <h4>ShortLink: </h4>
-                                    <a href='/${shortUrl}' class="short-url">/${shortUrl}</a>
-                                  </div>`)
+            <div>
+              <h4>Title: </h4>
+              <p>${urls[urls.length-1].title}</p>
+            </div>
+            <div>
+              <h4>ShortLink: </h4>
+              <a href= ${shortUrl} id ='test-button' class="short-url">/${shortUrl}</a>
```

Comment:
Is this id left over from trying to get the incrementer functionality working? If not, could it have a more semantic name?